### ISSUE TITLE
Config file inside archive is incorrect

### ISSUE THREAD

**Issue by sverhoeven** [i-VRESSE/workflow-builder#65]
I created a workflow which in text preview looked like

```toml
molecules = [
  'e2a_1F3G.pdb',
]
run_dir = 'x'

[flexref]

seg_sta_1_1 = 27
seg_end_1_1 = 35
fle_sta_1_1 = 140
fle_end_1_1 = 165
```

However in the archive it looked like 
```toml
molecules = [
  'e2a_1F3G.pdb',
]
run_dir = 'x'

[flexref]

seg = [
  [
    { sta = 27, end = 35 },
  ],
]
fle = [
  [
    { sta = 140, end = 165 },
  ],
]

```

I expected the workflow.cfg file in the archive look the same as the in the text preview.

**Comment by joaomcteixeira** [comment]
Just a reminder, haddock3 `cfg` files need an empty line after multiline lists.

```toml
molecules = [
  'e2a_1F3G.pdb',
]

# the empty line above this comment is necessary
run_dir = 'x'

[flexref]

seg_sta_1_1 = 27
seg_end_1_1 = 35
fle_sta_1_1 = 140
fle_end_1_1 = 165
```

**Comment by sverhoeven** [comment]
Fixed in #66
